src/test/java/easytests/core/services/UsersServiceTest.java

```diff
@@ -38,66 +39,44 @@
 
     private UsersSupport usersSupport = new UsersSupport();
 
-    private UserModelInterface mapUserModel(UserEntity userEntity) {
-        final UserModelInterface userModel = new UserModel();
-        userModel.map(userEntity);
-        return userModel;
-    }
-
-    private UserEntity mapUserEntity(UserModelInterface userModel) {
-        final UserEntity userEntity = new UserEntity();
-        userEntity.map(userModel);
-        return userEntity;
-    }
+    @Captor
+    private ArgumentCaptor<List<UserModelInterface>> usersModelsListCaptor;
 
-    private List<UserEntity> getUsersEntities() {
+    private List<UserEntity> getUsersFixturesEntities() {
         final List<UserEntity> usersEntities = new ArrayList<>(2);
-        final UserEntity userEntityFirst = this.usersSupport.getEntityMock(
-                1,
-                "FirstName1",
-                "LastName1",
-                "Surname1",
-                "email1@gmail.com",
-                "hash1",
-                true,
-                1
-        );
-        final UserEntity userEntitySecond = this.usersSupport.getEntityMock(
-                2,
-                "FirstName2",
-                "LastName2",
-                "Surname2",
-                "email2@gmail.com",
-                "hash2",
-                false,
-                2
-        );
-        usersEntities.add(userEntityFirst);
-        usersEntities.add(userEntitySecond);
+        usersEntities.add(this.usersSupport.getEntityFixtureMock(0));
+        usersEntities.add(this.usersSupport.getEntityFixtureMock(1));
         return usersEntities;
     }
 
-    private List<UserModelInterface> getUsersModels() {
+    private List<UserModelInterface> getUsersFixturesModels() {
         final List<UserModelInterface> usersModels = new ArrayList<>(2);
-        for (UserEntity userEntity: this.getUsersEntities()) {
-            usersModels.add(this.mapUserModel(userEntity));
-        }
+        usersModels.add(this.usersSupport.getModelFixtureMock(0));
+        usersModels.add(this.usersSupport.getModelFixtureMock(1));
         return usersModels;
     }
 
+    private void assertEquals(List<UserModelInterface> first, List<UserModelInterface> second) {
+        Assert.assertEquals(first.size(), second.size());
+        Integer i = 0;
```

Comment:
Are we really should to use Integer everywhere instead of int? (It's just a cycle, so are we realy need this heavyweight Integer as index of iterations?) It has the smaller size (integer is 16 byte and int is 4 byte), so we could gain more free memory.

Reply:
@Ielay 
It is Java! ;) We don't care about memory int tests.
But, we could use simple int type, I just don't want to confuse anyone with int.